consensus/parlia/fusionfork.go

```diff
@@ -0,0 +1,226 @@
+package parlia
+
+import (
+	"container/heap"
+	"context"
+	"math"
+	"math/big"
+	"sort"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/core/systemcontracts"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/internal/ethapi"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+// init contract
+func (p *Parlia) initFusionContract(state *state.StateDB, header *types.Header, chain core.ChainContext,
+	txs *[]*types.Transaction, receipts *[]*types.Receipt, receivedTxs *[]*types.Transaction, usedGas *uint64, mining bool,
+) error {
+	// method
+	method := "initialize"
+	// contracts
+	contracts := []string{
+		systemcontracts.StakeHubContract,
```

Comment:
I guess we have some other contract need to be initialized as well, please do double check.

Reply:
Sure. I will add a TODO here and check in the end.

Reply:
Let us add more now.

Reply:
Done